samples/generated_samples/snippet_metadata_google.cloud.osconfig.v1.json

```diff
@@ -8,7 +8,7 @@
     ],
     "language": "PYTHON",
     "name": "google-cloud-os-config",
-    "version": "0.1.0"
+    "version": "1.15.0"
```

Comment:
This is a major version bump. Is this correct? Why was this 0.1.0 previously?

Reply:
It seems like this was "1.14.0" prior to https://github.com/googleapis/python-os-config/pull/261/files, so I', taking the 0.1.0 to not be correct. I'll go ahead and allow a merge.